tools/stonith_admin.c

```diff
@@ -528,6 +528,12 @@ main(int argc, char **argv)
         target = options.unregister_level;
     }
 
+    if (options.timeout > INT_MAX / 1000 || options.timeout < INT_MIN / 1000) {
+        printf("stonith_admin: Integer value \"%d\" for -t out of range\n", options.timeout);
+        exit_code = CRM_EX_USAGE;
+        goto done;
+    }
```

Comment:
Guess we should do the error output more inline with how it is done for other errors.
Is there a sense in a negative timeout?